src/schema/schema_drop.c

```diff
@@ -90,6 +90,46 @@ __drop_index(
     return (ret);
 }
 
+/*
+ * __drop_layered --
+ *     WT_SESSION::drop for a layered table.
+ */
+static int
+__drop_layered(
+  WT_SESSION_IMPL *session, const char *uri, bool force, const char *cfg[], bool check_visibility)
+{
+    WT_DECL_ITEM(ingest_uri_buf);
+    WT_DECL_RET;
+    const char *ingest_uri, *tablename;
+
+    WT_ASSERT(session, WT_PREFIX_MATCH(uri, "layered:"));
+
+    WT_RET(__wt_scr_alloc(session, 0, &ingest_uri_buf));
+
+    tablename = uri;
+    WT_PREFIX_SKIP_REQUIRED(session, tablename, "layered:");
+    WT_ERR(__wt_buf_fmt(session, ingest_uri_buf, "file:%s.wt_ingest", tablename));
+    ingest_uri = ingest_uri_buf->data;
+
+    /*
+     * Drop the ingest table. We don't drop the stable component, since it's not a real table.
+     */
```

Comment:
Can we add a TODO here to call PALI to drop the stable table?

Reply:
Good call - done in `d37b19185`.